composer.json

```diff
@@ -72,6 +72,7 @@
     },
     "require-dev": {
         "aws/aws-sdk-php": "~3.0",
+        "iron-io/iron_mq": "~4.0.1",
```

Comment:
@GrahamCampbell Thanks, I will add it to suggests block but what do you mean by "BOTH" composer.json files?